support/build/plugin-lodash-import-rename.js

```diff
@@ -0,0 +1,24 @@
+import _ from 'lodash';
```

Comment:
Yes, it changes our `import foo from 'lodash'` to use the correct ES version when performing the rollup.